src/wp-includes/blocks/index.php

```diff
@@ -12,6 +12,54 @@
 require BLOCKS_PATH . 'widget-group.php';
 require BLOCKS_PATH . 'require-dynamic-blocks.php';
 
+/**
+ * Registers core block types styles.
```

Comment:
It's worth clarifying why this function exists, compared to `register_block_style_handle()`.
```suggestion
 * Registers core block style handles.
 *
 * While {@see register_block_style_handle()} is typically used for that, the way it is
 * implemented is inefficient for core block styles. Registering those style handles here
 * avoids unnecessary logic and filesystem lookups in the other function.
```